fladle-plugin/src/main/java/com/osacky/flank/gradle/YamlWriter.kt

```diff
@@ -140,6 +142,18 @@ internal class YamlWriter {
     }
   }
 
+  private fun StringBuilder.appendAdditionalProperty(property: Property<String>) {
+    if (property.isPresent) {
+      property.get()
+        .normalizeLineEnding()
+        .split("\n")
+        .map { "  $it" }
+        .forEach { appendln(it) }
+    }
+  }
+
+  private fun String.normalizeLineEnding() = replace("\r\n", "\n")
```

Comment:
why would there be `\r\n` line endings coming from the flank configuration?

Reply:
I think this is due to Windows support

Reply:
Right, but these strings are not coming from the filesystem. They strings declared in the buildscripts as configuration options on the fladle extension. Or am I mistaken?

Reply:
Yeah, this is just for supporting Windows properly. These options come from fladle extension, however, I guess someone could input them using "enter" on the keyboard and this is just to make sure to handle options properly

Reply:
I appreciate your trying to cover all the edge cases but I don't think enter on the keyboard would add a `\r\n` on windows. If this were the case we would need to add this to all java strings everywhere in this entire plugin. This only happens when things are coming from the filesystem. Please remove this unless you can show an example or write a test for this.

Reply:
sure, removed!